.github/workflows/rust.yml

```
@@ -123,3 +123,57 @@ jobs:
           cd /repo/contracts &&\
           cargo check &&\
           cargo clippy -- -D warnings"
+
+  relay_run_interface_tests:
+    name: Relay Run Interface Tests
+    runs-on: ubuntu-latest-8cores-32GB
+    needs: [get_projectserum_version, build_wrapped_anchor_image]
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@44c2b7a8a4ea60a981eaca3cf939b5f4305c123b # v4.1.5
+
+      - name: Cache cargo target dir
+        uses: actions/cache@0c45773b623bea8c8e75f6c82b208c3cf94ea4f9 # v4.0.2
+        with:
+          path: contracts/target
+          key: ${{ runner.os }}-v2-cargo-build-target-${{ hashFiles('**/Cargo.lock') }}
+
+      - name: cache docker build image
+        id: cache-image
+        uses: actions/cache@0c45773b623bea8c8e75f6c82b208c3cf94ea4f9 # v4.0.2
+        with:
+          fail-on-cache-miss: true
+          path: contracts/docker-build.tar
+          key: ${{ runner.os }}-docker-pnpm-build-${{ needs.get_projectserum_version.outputs.projectserum_version }}-${{ hashFiles('**/Cargo.lock') }}
+
+      - name: load cached image
+        run: |
+          docker load --input docker-build.tar
+
+      - name: Setup go
+        uses: actions/setup-go@cdcb36043654635271a94b9a6d1392de5bb323a7 # v5.0.1
+        with:
+          go-version-file: "./integration-tests/go.mod"
+          check-latest: true
+          cache-dependency-path: "./integration-tests/go.sum"
+
+      - name: Install gotestloghelper
+        run: go install github.com/smartcontractkit/chainlink-testing-framework/tools/gotestloghelper@latest
+
+      - name: Install Solana CLI
+        run: ../scripts/install-solana-ci.sh
```

```suggestion
        run: ./scripts/install-solana-ci.sh
```

Workflow steps by default run from the root.